setup-environment-internal

```diff
@@ -234,7 +234,7 @@ if [ -d "${MANIFESTS}"/factory-keys ]; then
         ln -sf "${MANIFESTS}"/factory-keys/uefi conf/factory-keys/uefi
     fi
     # Link default TI K3 RoT keys if not set by the user
-    if  -d "${MANIFESTS}"/factory-keys/platform -a ! -d "conf/factory-keys/platform" ]; then
+    if [ -d "${MANIFESTS}"/factory-keys/platform -a ! -d "conf/factory-keys/platform" ]; then
```

Comment:
shellcheck suggests this isn't the best syntax:
```
In setup-environment-internal line 229:
    if [ -d "${MANIFESTS}"/factory-keys/tf-a -a ! -d "conf/factory-keys/tf-a" ]; then
                                             ^-- SC2166 (warning): Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.
```

Maybe this should be updated in a separate patch.

Reply:
Yeah, change to the suggested one as that would be indeed better.